packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -247,6 +250,13 @@ export class MigrationActions {
     const from = path.join(projectRoot, 'cypress', 'integration')
     const to = path.join(projectRoot, 'cypress', 'e2e')
 
+    this.ctx.update((coreData) => {
+      coreData.migration.flags = {
+        ...coreData.migration.flags,
+        shouldAddCustomE2eSpecPattern: true,
```

Comment:
Nit: we usually write this as `E2E`, not `E2e`, in this code base from what I can see.